release.yml

```diff
@@ -16,7 +16,7 @@ variables:
   functionalTests: "**/*FunctionalTests/*.csproj"
   buildConfiguration: 'Release'
   major: 3
-  minor: 4
+  minor: 5
```

Comment:
Are we going to upgrade version to v3.5?

Reply:
Yes, since cda default templates are supported, we could upgrade the version to v3.5. After discussing with Yue, I will undo it, and create a new pr to upgrade version.